### ISSUE TITLE
Are KSO and KS FC earned or unearned runs?

### ISSUE THREAD

**Issue by AloisSeckar** [AloisSeckar/WBSC-Scoring#99]
It is not exactly clear from the manual - it feels they should be treated same as KS PB, but needs verification...

**Comment by lucafano04** [comment]
Well Ks/O and Ks/FC are ER
For a K considered a ER (when safe at first obviously) it must be a Ks WP unless a Ks PB is a decisive error of the catcher, a Ks 2E3 is a "Big E" error of the first basement, as Ks #T...

**Comment by AloisSeckar** [comment]
My doubt in this matter was based on the fact, that KS/O and KS/FC can only happen due to dropped 3rd strike. And since catcher must also be able to throw to another base during this play, the ball must be close to his reach. This means most of the time it would be his passed ball, not the wild pitch (KS WP usually ends up on the backstop). So why pitcher should be blamed for catcher's misplay?

**Comment by lucafano04** [comment]
> My doubt in this matter was based on the fact, that KS/O and KS/FC can only happen due to dropped 3rd strike. And since catcher must also be able to throw to another base during this play, the ball must be close to his reach. This means most of the time it would be his passed ball, not the wild pitch (KS WP usually ends up on the backstop). So why pitcher should be blamed for catcher's misplay?

Indeed, if it is a Ks/PB it's not a ER instead if it is a KS/FC the chatcher can has choosen to play to other runners, the pitcher an error is missing but from the rules:

> **18.3.2** The Official Scorer to determine earned runs shall reconstruct the inning without passed balls or errors (excluding catcher’s interference). When the Official Scorer credits a fielder with an error, the pitcher will always be given the benefit of the doubt with respect to determining what base or bases a runner or runners would have reached on a play without errors. The Official Scorer shall recognise that an error by the pitcher will be treated the same as an error by another fielder
And:
> **18.3.3** THE OFFICIAL SCORER SHALL NOT RECORD AN EARNED RUN WHEN:
> > **18.3.3.1** A batter-runner reaches first base as follows:
> > > **18.3.3.1.1** For an error of a player to the defence.

> > > **18.3.3.1.2** For a hit or other play after his time at bat was prolonging by a foul fly that was not caught by a fielder and the Official Scorer credited the fielder with an error for such fault.

> > > **18.3.3.1.3** For defensive interference or obstruction
When a batter is awarded first base on catcher’s interference and scores, the run will not count as an earned run. However, the Official Scorer in this case will not assume hat such batter would have been out in the absence of defensive interference, since he did not have the opportunity to complete the at-bat and it is unknown how it would have ended.

> > **18.3.3.2** A run is scored by a runner whose time on bases was prolonged by an error and who could have been put out on the play without the error.

> > **18.3.3.3** A run was scored as a runner advanced one or more bases on an error, passed ball, defensive interference, or obstruction and in the judgment of the Official Scorer that run would not have been scored without the aid of the misplay.

**Comment by AloisSeckar** [comment]
Answered by Dennis Duin / Linda Steijger from WBSC Europe:

_The situation where a KS O2 is scored for the batter-runner, should be compared with a O6 when a former runner is putout 64. The catcher corrected dropping the third strike by making an out at home plate.

**A run scored by this runner can still be an earned run.** It cannot be compared to a KS PB._

--

Therefore, no action needed. No need for _"KS O2 cannot be earned run"_ restrictions.
